ibm/7.0/ha/fgt.tf

```diff
@@ -106,15 +106,15 @@ data "template_file" "userdata" {
   template = file("${var.bootstrap}")
 
   vars = {
-    license_file = "${file("${var.license}")}"
-    hapriority   = "255"
-    apikey       = "${var.ibmcloud_api_key}"
-    port1_ip     = "${var.fgtaport1}"
-    port1_mask   = "${var.fgtaport1mask}"
-    port2_ip     = "${var.fgtaport2}"
-    port2_mask   = "${var.fgtaport2mask}"
-    port3_ip     = "${var.fgtaport3}"
-    port3_mask   = "${var.fgtaport3mask}"
+    license_file    = "${file("${var.license}")}"
```

Comment:
surprised these weren't caught in the fmt.

Reply:
@Joel-Cripps I'm not sure what the change request is ?

Reply:
@Joel-Cripps I think I understand what you meant. I do see the fmt command didn't adjust the values inside the the data resource, and I see there is some inconsistencies. I will review to see if there are other areas that need attention, and I will update the PR